.github/workflows/enteprise.yml

```diff
@@ -78,55 +84,29 @@ jobs:
     if: github.actor != 'dependabot[bot]'
     needs:
       - docker
-    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         molecule_distro:
           - image: ubuntu:22.04
           - image: rockylinux:9.2
         scenario:
-          - name: elasticsearch
           - name: pki
+          - name: elasticsearch
         include:
           - scenario:
               name: identity
             molecule_distro:
               image: rockylinux:9.2
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          # avoid auth conflict when cloning DTAS during verify step
-          persist-credentials: false
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        id: setup-python
-        with:
-          python-version-file: '.python-version'
-          cache: pipenv
-
-      - name: Install python packages
-        uses: Alfresco/alfresco-build-tools/.github/actions/pipenv@v3.4.0
-        with:
-          python-version: ${{ steps.setup-python.outputs.python-version }}
-
-      - name: Install and cache ansible galaxy dependencies
-        uses: ./.github/actions/galaxy
-        with:
-          cache-name: enterprise
-
-      - name: Setup workspace
-        uses: ./.github/actions/setup-workspace
-
-      - name: Run tests
-        env:
-          MOLECULE_ROLE_IMAGE: ${{ matrix.molecule_distro.image }}
-          NEXUS_USERNAME: ${{ secrets.nexus_username }}
-          NEXUS_PASSWORD: ${{ secrets.nexus_password }}
-          CLONE_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
-        run: pipenv run molecule test -s ${{ matrix.scenario.name }}
+    uses: ./.github/workflows/docker.yml
+    with:
+      scenario: ${{ matrix.scenario.name }}
+      os_distribution: ${{ matrix.molecule_distro.image }}
+      dtas_version: ${{ needs.docker.outputs.dtas_version }}
```

Comment:
it's defined as env in the global level, you should be able to reference it here without the need of setting an output

Reply:
the env context is not available in `jobs.<job_id>.with`